### ISSUE TITLE
Scan QR broken on GrapheneOS Pixel 5

### ISSUE THREAD

**Issue by DanConwayDev** [MutinyWallet/mutiny-web#552]
**test**: Send > Scan QR
**expected behaviour:** prompt to allow access to camera, select 'allow', image from camera displays to scan QR code.
**actual behaviour**: black screen with 'Paste Something' and 'Cancel' buttons at the bottom.

**failed test scenarios**:
* Vadadium PWA installed with camera 'ask every time' permissions
* Vadadium PWA installed with camera 'allowed' permissions
* Vadadium with camera 'ask every time' permissions
* Vadadium with camera 'allowed' permissions

**test conditions**:
Pixel 5 running Graphene OS Andriod 13 Pixel 5 Build/TQ3A.23.0901.001 Vanadium version 117.0.5938.60

**Comment by DanConwayDev** [comment]
It is working on the same device with the APK version

**Comment by TonyGiorgio** [comment]
I use graphene and know a few others that do as well. Haven't seen any camera problems with them yet. Has it ever worked? Does manually setting the permissions in settings not work?

**Comment by DanConwayDev** [comment]
No. I successfully tested that the camera worked on other websites. I have fully switched over to the APK version now so its not bothering me anymore. Just something to bear in mind if other users report it. I'll leave it up to you whether you want to close the issue.

**Comment by NickRHill** [comment]
Same for me. With the web version on a Cubot X19 phone I get a dark green screen.

**Comment by TonyGiorgio** [comment]
I've heard that using the web browser version (not the PWA) to do the camera part will generate the prompt.

**Comment by NickRHill** [comment]
It works on the Linux desktop version of Chromium.

I had the prompt for the camera on Android. I can see it is allowed in the settings on Chrome for Android. I tried revoking permission, it asked again, I accepted and I still get dark green screen. Can't see anything from the camera.

**Comment by benthecarman** [comment]
You can always download the app

**Comment by NicolaLS** [comment]
We seem to have the same/or at least related problem in the [BitBoxApp](https://github.com/digitalbitbox/bitbox-wallet-app) (Android app on GrapheneOS) since we switched to [qr-scanner](https://github.com/nimiq/qr-scanner).

Users reported that the app crashes. One user reported he saw a message about _native code debugging_ "native code debugging required". See [this](https://grapheneos.org/usage#banking-apps) for reference.
